styles.css

```diff
@@ -182,10 +136,9 @@ body {
 }
 
 .subtitle-container {
-  width: 100%;
+  width: 110%;
   font-size: 13px;
   font-weight: 600;
-  line-height: 16px;
   margin-top: 0;
   border-top: 0;
```

Comment:
- [ ] KIndly make sure your page doesn't overflow as you can see here on your design for the mobile version :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193144184-6b26c3bb-73ae-4028-beb0-33efeea0fbd0.png)